modules/serverless-jaeger-config.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * /serverless/serverless-tracing.adoc
+// */jaeger/jaeger_config/serverless-jaeger-integration.adoc
+
+[id="serverless-jaeger-config_{context}"]
+= Configuring Jaeger for use with {ServerlessProductName}
+
+.Prerequisites
+
+To configure Jaeger for use with {ServerlessProductName}, you will need:
+
+* Cluster administrator permissions on an {product-title} cluster.
+* A current installation of {ServerlessOperatorName} and Knative Serving.
+* A current installation of the Jaeger Operator.
+
+.Procedure
+
+. Create and apply the Jaeger custom resource:
++
+----
+$ cat <<EOF | oc apply -f -
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: jaeger
+EOF
+----
+
+. Enable tracing for Knative Serving, by editing the `KnativeServing` resource and adding a YAML configuration for tracing.
++
+.Tracing YAML example
+----
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+spec:
+  config:
+    tracing:
+      sample-rate: "0.1"
+      backend: zipkin <1>
+      zipkin-endpoint: http://jaeger-collector.<JAEGER_NAMESPACE>.svc.cluster.local:9411/api/v2/spans <2>
+      debug: "true"
+----
++
+<1> The backend must be `zipkin`.
+<2> The `zipkin-endpoint` must point to your `jaeger-collector` service endpoint. To get this endpoint, substitute the namespace where the Jaeger Operator is installed.
++
+.Example Jaeger service endpoint
+----
+http://jaeger-collector.openshift-operators.svc.cluster.local:9411/api/v2/spans
+----
+
+.Verification  steps
+Access the Jaeger web console to see tracing data. You can access the Jaeger web console by using the `jaeger-collector` service endpoint.
+
+. Get the `jaeger-collector` service endpoint:
++
+----
+$ oc get route
+----
+. Open the endpoint address in your browser. The service endpoint should be in the format `pass:c[https://jaeger-<namespace>.apps.<snip>]`.
```

Comment:
Here's an example command:

```console
$ oc get route jaeger
NAME     HOST/PORT                         PATH   SERVICES       PORT    TERMINATION   WILDCARD
jaeger   jaeger-default.apps.example.com          jaeger-query   <all>   reencrypt     None
```

Let's change the command to be "oc get route jaeger" or "oc -n default get route jaeger" if the default namespcae is where the Jaeger resource is applied (see comments above).